google/cloud/aiplatform/jobs.py

```diff
@@ -1046,6 +1104,29 @@ def network(self) -> Optional[str]:
         self._assert_gca_resource_is_available()
         return self._gca_resource.job_spec.network
 
+    def _get_web_access_uris(self):
+        """Helper method to get the web access uris of the custom job"""
+        self._web_access_uris = self._gca_resource.web_access_uris
```

Comment:
Based on the usage of this method, it seems like we can remove the `_web_access_uris` property and return `web_access_uris` in this method.

So usage is:
```Python
web_access_uri = self._get_web_access_uris()

for worker, uri in web_access_uris.items():
  ...
```